daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -1388,7 +1388,9 @@ private[lf] object Speedy {
     */
   private[speedy] final case class SpeedyHungry(result: SResult)
       extends RuntimeException
-      with NoStackTrace
+      with NoStackTrace {
+    override def toString = s"SpeedyHungry($result)"
```

Comment:
how is this different from what the autogenerated `toString` gives you?

Reply:
Here there is no autogenerated `toString`. The one from `Throwable` has precedence